examples/stateful-workloads/variables.tf

```diff
@@ -18,16 +18,7 @@ variable "node_pools" {
     zones      = optional(list(string))
     os_type    = string
   }))
-  default = {
-    # This is an example of a node pool for a stateful workload with minimal configuration
-    stateful = {
```

Comment:
i think we can keep this line so that the tests we are running test for something i'll revert this part